.cache

```diff
@@ -0,0 +1 @@
+{"access_token": "BQBOFvTjRqI66SdCAo12CxZIMju5q7CUVspG68WVrnZLPdtcd4wZVq5Ky9rdHrAfCUVlS1zus9tbzu-qP9vPmZ9tAYAKbQd26A7d7UReHyzcHSqLq60", "token_type": "Bearer", "expires_in": 3600, "expires_at": 1703941566}
```

Comment:
your token is exposed here

Reply:
It should be alright now since i updated the client access.